src/main/java/org/geysermc/resources/state/type/PressurePlateSignalMapper.java

```diff
@@ -0,0 +1,14 @@
+package org.geysermc.resources.state.type;
+
+import org.geysermc.resources.Pair;
+import org.geysermc.resources.state.StateMapper;
+import org.geysermc.resources.state.StateRemapper;
+
+@StateRemapper(value = "powered", blockRegex = ".*_pressure_plate.?$")
+public class PressurePlateSignalMapper extends StateMapper<Integer> {

```

Comment:
Needs to be named to PressurePlatePoweredMapper. Naming convention is `(block)(java state)Mapper`.